Ferlab-Ste-Justine/clin-portal-ui
# fix: CLIN-1991 Added noDataInputOption in config for range filters

# FEAT : [NoData] Enlever le no data des facettes NUMÉRIQUES des métriques du donor

- closes #CLIN-1991 CLIN-1913
## Description
Remove the noData checkbox for numerical donors range facets

[JIRA LINK]

Acceptance Criterias
- In donors numerical facets, the noData checkbox is not visible

## Validation


## Screenshot
### Before
![image](https://github.com/Ferlab-Ste-Justine/clin-portal-ui/assets/10870599/0f2ef4a0-c4bf-4eb2-9434-dc13c0578d8b)

### After
![image](https://github.com/Ferlab-Ste-Justine/clin-portal-ui/assets/10870599/97038a40-a32b-4371-ae3a-9a628949eb06)


## QA

Steps to validate
- go in the variants page of a patient
- open occurence menu
- open the Genome Quality filter
- No data should not be visible

## Mention

@kstonge @luclemo
